queries.sql

```diff
@@ -0,0 +1,55 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * from animals WHERE name like '%mon';
+SELECT name from animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name from animals WHERE neutered = true;
+SELECT name from animals WHERE escape_attempts < 3;
+SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = true;
+SELECT *
+FROM animals
+WHERE NOT name = 'Gabumon';
+SELECT * from animals WHERE weight_kg between 10.4 and 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+
+
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+COMMIT;
+SELECT * FROM animals;
+
+
+
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+
+
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT savepoint1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO savepoint1;
```

Comment:
- [ ] If you count all the queries based on the project requirement, you will discover they are about 9 queries in total. Therefore you are expected to add about nine screenshots. Currently, your screenshot is not clear as to which query they contain. I will suggest you add the particular query as a label above the screenshot e.g 👇🏾  And please I suggest you use the snipping tool to take smaller screenshots. [Learn how to here.](https://support.microsoft.com/en-us/windows/open-snipping-tool-and-take-a-screenshot-a35ac9ff-4a58-24c9-3253-f12bac9f9d44)

**Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.
Verify that change was made and persists after commit.** 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/225575546-894e32dd-1932-42d1-9e78-7c0b6769fca5.png)

- And ensure that the query commands are visible, unlike your screenshot at the point where I put a red mark. 
- Kindly follow this format for all other queries so it is easy for the next reviewer to see and understand which screen is for which query. 

**_How many animals are there?_** 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/225577882-802917d6-045c-4ee2-b622-0e1b0cd30086.png)

_**How many animals have never tried to escape?**_ 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/225577882-802917d6-045c-4ee2-b622-0e1b0cd30086.png)

_**What is the average weight of animals?**_ 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/225577882-802917d6-045c-4ee2-b622-0e1b0cd30086.png)